scripts/ci-push.sh

```diff
@@ -4,11 +4,20 @@ set -o errexit -o pipefail
 
 source ./scripts/ci-login.sh
 
-./scripts/build-site.sh
-./scripts/sync-and-test-bucket.sh
 
-# Wait for in-progress jobs to complete before proceeding.
-node ./scripts/await-in-progress.js
-
-./scripts/run-pulumi.sh update
+if [[ "$GITHUB_WORKFLOW" == "Build and deploy" ]]; then
+    # Wait for in-progress jobs to complete before proceeding.
+    ./scripts/build-site.sh
+    ./scripts/sync-and-test-bucket.sh
+    node ./scripts/await-in-progress.js
+    ./scripts/run-pulumi.sh update
+elif [[ "$GITHUB_WORKFLOW" == "Build and deploy new account" ]]; then
+    # temporarily setting site to build in preview mode, to prevent search bots from indexing
+    # until we cut over DNS to start routing to the new account.
+    ./scripts/build-site.sh preview
+    ./scripts/sync-and-test-bucket.sh
```

Comment:
One thing to keep in mind here is that this new account probably has the default bucket quota, which I believe is 100 buckets. The current production account allows up to 500, I think. We clean these up nightly in another workflow that won't be run here (yet), so we'll probably hit that limit fairly pretty quickly in the new account. Manual cleanup in the console is probably fine (and safer) for now, so I mainly just wanted to call that out as something to be aware of.

Reply:
good call out